settings.gradle

```diff
@@ -8,5 +8,5 @@
  */
 
 rootProject.name = 'varadhi'
-include('entities', 'spi', 'common', 'core', 'messaging', 'pulsar', 'consumer', 'controller', 'authz', 'server')
+include('entities', 'spi', 'common', 'core', 'messaging', 'pulsar', 'consumer', 'controller', 'authz', 'server', 'rateLimiter')
```

Comment:
lets use the rate_limiter or rate-limiter. lets not use camelCase. I have usually seen hyphens being used.

for eg:
implementation group: 'org.slf4j', name: 'slf4j-api', version: "$slf4j_version"

com.google.cloud:google-cloud-core

Reply:
renamed to `ratelimiter` as adding `rate-limiter` in package name was breaking module-info

Reply:
done, renamed module to rate-limiter